packages/pointer_interceptor/pointer_interceptor/lib/src/dummy_pointer_interceptor_ios.dart

```diff
@@ -0,0 +1 @@
+// TODO(louisehsu): Replace this empty file with PointerInterceptorIOS Implementation.
```

Comment:
This actually can't be reverted, I think
Since we're trying not to break any existing implementations, I think there are cases where they are using the pointer_interceptor with ios. Currently, the ios implementation in the old pointer_interceptor package does nothing. I wanted to keep the existing no-op behaviour, but also allow them to opt in for the ios implementation
Basically since this file is empty, without the pointer_interceptor_ios plugin, on it will use the NoopPointerInterceptor as a default. However, if they do use the new ios package, it will use that implementation because that's the only iOS implementation available. And since I think that theres no way to override a platform implementation with another package (https://github.com/flutter/flutter/issues/80374), this is just a slightly hacky way of doing it until we make the new ios implementation the default.

Reply:
Since we eventually will likely to have the new behavior to be the default, maybe we should print a message to warn users that the no-op behavior will be deprecated soon, and the developer should remove the widget if they want no-op in mobile. (and introduce how to opt-in the new plugin probably)

Reply:
Sounds good! Ill add that.

Reply:
> I wanted to keep the existing no-op behaviour, but also allow them to opt in for the ios implementation
> Basically since this file is empty, without the pointer_interceptor_ios plugin, it will use the NoopPointerInterceptor as a default.

You can just not have an `ios` entry in the app-facing package at all to get this behavior.

Reply:
Looking at pub.dev, pointer_interceptor is only used in a very small number of published packages; presumably most of the use is in applications. Given that, I would just make adding the iOS implementation a breaking version change, rather than have a warning->deprecation->replacement cycle (which would need to end with a breaking version change anyway).

Reply:
Okay! If we're okay with breaking changes, Ill go ahead and refactor pointer_interceptor_ios to be part of the package. it's definitely cleaner, anyways.